hieradata/common.yaml

```diff
@@ -79,6 +79,13 @@ backup::offsite::jobs:
     hour: 8,
     minute: 13,
     # No encryption because of size and sensitivity
+  'govuk-cdn-logs':
+    sources: "/data/backups/logs-cdn-1.management.%{hiera('app_domain')}/*"
+    destination: "rsync://%{hiera('backup::offsite::dest_host')}//srv/backup-cdn-logs/"
+    weekday: 6
```

Comment:
We don't backup our Graphite data daily because of the size of the data. Is there a good reason not to do daily backups of our processed CDN logs?